ledger/src/sigverify_shreds.rs

```diff
@@ -295,7 +296,21 @@ pub fn verify_shreds_gpu(
         elems_from_buffer(&pubkeys),
         elems_from_buffer(&merkle_roots),
     ];
-    elems.extend(batches.iter().map(|batch| perf_libs::Elems {
+    // `BytesPacketBatch` cannot be directly used in CUDA. We have to retrieve
+    // and convert byte batches to pinned batches.
+    let mut pinned_batches = Vec::new();
```

Comment:
I was thinking if I should do `with_capacity(batches.len())` or come up with some heuristic how much to allocate, but not sure if it's worth it.